randovania/games/dread/json_data/Hanubia.txt

```diff
@@ -973,24 +860,28 @@ Extra - asset_id: collision_camera_015
   * EMMI Door to EMMI Zone Exit East/Dock to Total Recharge Station East (Upper)
   * Extra - actor_name: dooremmy_002
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
-  > Tile Group (POWERBOMB) 2
-      Trivial
+  > Dock to EMMI Zone Exit East (Lower)
+      Lay Power Bomb
 
 > Dock to EMMI Zone Exit East (Lower); Heals? False
   * Layers: default
   * EMMI Door to EMMI Zone Exit East/Dock to Total Recharge Station East (Lower)
   * Extra - actor_name: dooremmy_003
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
+  > Dock to EMMI Zone Exit East (Upper)
+      All of the following:
+          Lay Power Bomb
+          Speed Booster or Simple IBJ or Use Spin Boost
   > Total Recharge
       Trivial
-  > Tile Group (POWERBOMB) 1
-      Trivial
 
 > Dock to Escape Room 2; Heals? False
   * Layers: default
   * Open Passage to Escape Room 2/Dock to Total Recharge Station East
-  > Tile Group (POWERBOMB) 3
-      Trivial
+  > Dock to EMMI Zone Exit East (Lower)
+      Any of the following:
+          After Escape Sequence
+          Power Bombs ≥ 4 and Lay Power Bomb
```

Comment:
Why 4? There's only 2 blocks before the lower door, 1 of which is breakable from the previous escape room.